### PR TITLE
l10n: uk: update core translation

### PR DESCRIPTION
**Pull request by kasya**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -2002,114 +2002,116 @@ msgstr ""
 
 #: wt-status.c:242
 msgid "  (use \"git add/rm <file>...\" to update what will be committed)"
-msgstr ""
+msgstr "  (використовуйте \"git add/rm <файл>…\" для оновлення того, що буде закомічено)"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "  (використовуйте \"git add/rm <файл>...\" для оновлення того, що буде закомічено)"
```

File: po/uk.po [hunk 2]
```diff
@@ -2002,114 +2002,116 @@ msgstr ""
...
 
 #: wt-status.c:243
 msgid "  (use \"git restore <file>...\" to discard changes in working directory)"
-msgstr ""
+msgstr "  (використовуйте \"git restore <file>…\", щоб скасувати зміни в робочій директоріі)"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "  (використовуйте \"git restore <файл>...\", щоб скасувати зміни в робочій директорії)"
```

File: po/uk.po [hunk 3]
```diff
@@ -2002,114 +2002,116 @@ msgstr ""
...
 #: wt-status.c:245
 msgid "  (commit or discard the untracked or modified content in submodules)"
-msgstr ""
+msgstr "  (зробіть коміт або скасуйте невідстежуваний або змінений контент у підмодулях)"
 
 #: wt-status.c:256
 #, c-format
 msgid "  (use \"git %s <file>...\" to include in what will be committed)"
-msgstr ""
+msgstr "  (використовуйте \"git %s <файл>…\" для включення до того, що буде закомічено)"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "  (використовуйте \"git %s <файл>...\" для включення до того, що буде закомічено)"
```

File: po/uk.po [hunk 4]
```diff
@@ -2002,114 +2002,116 @@ msgstr ""
...
 
 #: wt-status.c:268
 msgid "both deleted:"
-msgstr ""
+msgstr "обидва видалено:"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "видалено обома:"
```

File: po/uk.po [hunk 5]
```diff
@@ -2002,114 +2002,116 @@ msgstr ""
...
 
 #: wt-status.c:276
 msgid "deleted by us:"
-msgstr ""
+msgstr "видалено нами:"
 
 #: wt-status.c:278
 msgid "both added:"
-msgstr ""
+msgstr "обидва додано:"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "додано обома:"
```

File: po/uk.po [hunk 6]
```diff
@@ -2002,114 +2002,116 @@ msgstr ""
...
 
 #: wt-status.c:280
 msgid "both modified:"
-msgstr ""
+msgstr "обидва змінено:"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "змінено обома:"
```

File: po/uk.po [hunk 7]
```diff
@@ -2121,23 +2123,23 @@ msgstr ""
...
 
 #: wt-status.c:1226
 msgid "  (use \"git merge --abort\" to abort the merge)"
-msgstr ""
+msgstr "  (використовуйте команду \"git merge --abort\", щоб перервати злиття)."
 
 #: wt-status.c:1230
 msgid "All conflicts fixed but you are still merging."
-msgstr ""
+msgstr "Всі конфлікти виправлено, але ви все ще продовжуєте злиття."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Усі конфлікти виправлено, але ви все ще продовжуєте злиття."
```

File: po/uk.po (hunk 1 above)
Comment (kasya) [reply po/uk.po]: I'm not sure what happened to those... Thanks!

**Pull request merged**